exercises/K-functions-parameters/exercise.js

```diff
@@ -1,9 +1,10 @@
 // Complete the function so that it takes input parameters
-function multiply() {
+function multiply(a,b) {
   // Calculate the result of the function and return it
+  return a * b;
 }
 
 // Assign the result of calling the function the variable `result`
 var result = multiply(3, 4);
-
+multiply()
```

Comment:
Calling function before console.log is useless, you can remove it and test it again and still get the results.